local_env/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
       METRICS_STORAGE_TOKEN: "myadmintoken"
       CORE_CLIENT_URL: "http://core:8080"
       DISABLE_SWAGGER_AUTHORIZATION: true
+      H2_FILE: ./data/testdb
```

Comment:
let's also change default in application.yaml. i'd be convenient if app will look inside /app/data by default